packages/connect-query/package.json

```diff
@@ -1,33 +1,34 @@
 {
   "name": "@connectrpc/connect-query",
   "version": "0.5.1",
+  "description": "TypeScript-first expansion pack for TanStack Query that gives you Protobuf superpowers.",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/connectrpc/connect-query-es.git",
     "directory": "packages/connect-query"
   },
-  "license": "Apache-2.0",
-  "sideEffects": false,
-  "type": "module",
-  "exports": {
-    ".": {
-      "module": "./dist/index.js",
-      "require": "./dist/index.cjs",
-      "import": "./dist/index.js"
-    }
-  },
-  "main": "./dist/index.cjs",
-  "files": [
-    "dist/**"
-  ],
   "scripts": {
-    "build": "tsup",
-    "clean": "rm -rf dist",
+    "clean": "rm -rf ./dist/*",
+    "build": "npm run build:cjs && npm run build:esm && npm run build:proxy",
```

Comment:
@timostamm Can we remove `tsup` as a dependency now?

Reply:
👍 https://github.com/connectrpc/connect-query-es/pull/180/commits/534f355d794a2485b2266605a82de3de86074874